cineast-core/src/main/java/org/vitrivr/cineast/core/db/cottontaildb/CottontailEntityCreator.java

```diff
@@ -358,20 +341,9 @@ public static Type mapAttributeType(AttributeDefinition.AttributeType type) {
   }
 
 
-  private boolean createIndexWithoutSchemaName(String entityName, String attribute, IndexType type, long txId) {
-    var fqn = CottontailWrapper.CINEAST_SCHEMA + "." + entityName;
-    final String indexName = fqn + ".idx_" + attribute + "_" + type.name().toLowerCase();
+  private void createIndex(String entityName, String attribute, IndexType type, long txId) {
+    final String indexName = entityName + ".idx_" + attribute + "_" + type.name().toLowerCase();
     final CreateIndex index = new CreateIndex(indexName, type).column(entityName + "." + attribute).txId(txId);
-    try{
-      this.cottontail.client.create(index);
-    }catch(StatusRuntimeException e){
-      if(e.getStatus().getCode()== Status.ALREADY_EXISTS.getCode()){
-        LOGGER.warn("Index {} was not created because it already exists", indexName);
-        return false;
-      }
-      throw e;
-    }
-    return true;
+    this.cottontail.client.create(index);
```

Comment:
I don't understand the diff here, I think something went wrong while merging. The functionality to create an Index for the ID seems to have been deleted, and the handling for if an index already exists seems also to have been deleted. Is this a merge-error (if so, I can fix that) or is this intentional?

Reply:
This looks like a merge error to me as well, git apparently got confused with the change order when I resurrected the dev branch.